Automattic/sensei
# Merge Course Lesson Wizard feature branch into trunk

### Testing instructions

Just test the plugin and the wizard, and make sure it seems to be working. Also ensure that all tests are passing.
